Dockerfile

```diff
@@ -148,6 +157,12 @@ RUN DEBIAN_FRONTEND=noninteractive \
       && apt clean \
       && rm -rf /var/lib/apt/lists/*
 
+# Install expect for the CLI integration tests.
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends expect || \
+  echo "Warning: expect installation failed, continuing without it" && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
is there no simpler way to install expect? this doing an rm -rf on lists, and _another_ apt-get clean seem like a bit much